addons/crm/models/crm_team.py

```diff
@@ -39,6 +40,7 @@ class Team(models.Model):
         ('user', 'Salesperson'),
         ('stage', 'Stage'),
     ], string='Grouping Method', default='day', help="How this channel's dashboard graph will group the results.")
+    incoming_emails = fields.Boolean('Incoming Emails', default=False)
```

Comment:
If you want to have an interface field to display / hide alias settings it should also deactivate the alias. Indeed if you configure the alias then uncheck incoming_emails your alias is still active and emails will still land in your pipeline. If it is requested to have this field then you should probably reset the alias name if we uncheck incoming_emails. Indeed deactivating aliases is done by settings its name to False.

Reply:
Finally it appears that this behaviour has been implemented in another task. I removed it.